public/app/core/services/backend_srv.ts

```diff
@@ -161,11 +161,8 @@ export class BackendSrv implements BackendService {
       }
     }
 
-    // Add device id header if not OSS build
-    if (config.buildInfo?.edition !== GrafanaEdition.OpenSource && this.deviceID) {
-      options.headers = options.headers ?? {};
-      options.headers['X-Grafana-Device-Id'] = `${this.deviceID}`;
-    }
+    options.headers = options.headers ?? {};
```

Comment:
Why did we we remove edition check here?

Reply:
We had a discussion around it and decided to life it. Here is the discussion:

@danielkenlee:
> Yes we should based on a previous discussion about enabling OSS to benefit and also ease the implementation by not limiting the change to Enterprise on-prem only.

@Jguer:
>  this allows us to lift the license check on the frontend